Adding aws creds

sekka1 opened the pull request:
* Adding in aws creds file

sekka1 commented:
Adding in the creds file doesnt seem to work.  It doesnt look like elastalert even uses the Auth class.

https://github.com/wercker/elastalert/pull/1/files#diff-247395ee788f793d37e74f424f117d17R33

the create_index.py does though.

hatchan commented on Dockerfile:
```diff
@@ -7,8 +7,7 @@ RUN easy_install pip
-RUN tar -xf v0.1.4.tar.gz -C /opt/
-RUN ls -l /opt
-WORKDIR /opt/elastalert-0.1.4
+RUN tar -xf v0.1.4.tar.gz -C /opt/ && mv v0.1.4.tar.gz elastalert
```
Think the mv part is not correct, since it is moving the tar file and then using that as the working directory.

sekka1 replied:
You are right...fixing this now.

hatchan commented on deployment/elastalert.template.yaml:
```diff
@@ -2,58 +2,76 @@
+    type: RollingUpdate
   template:
     metadata:
+      creationTimestamp: null
```
¯\\\_(ツ)\_/¯

sekka1 replied:
Removing

sekka1 commented:
Updated the rules selection so that it looks at the entire message instead of the "MESSAGE" field.  We changed the tagging on the ES logs a month or so ago and this field is gone now.

sekka1 commented:
Deployed to `staging`

sekka1 commented:
Deployed to `production`

sekka1 commented:
Tested that the alerting works by changing the rules config maps manually on a search term that would trigger.

Test alert message in slacK: https://wercker.slack.com/archives/chatbots/p1488905275044271

sekka1 commented:
@flenter can you take a look?
